File: TreasurersApp/Models/TransactionType.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace TreasurersApp.Models
{
    public partial class TransactionType
    {
        public TransactionType()
        {
            CashJournal = new HashSet<CashJournal>();
        }

        [JsonProperty("id")]
        public int TransactionTypeId { get; set; }

        [JsonProperty("transactionCategoryId")]
        public int TransactionCategoryId { get; set; }

        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }

        [JsonProperty("displayOrder")]
        public int DisplayOrder { get; set; }

        [JsonProperty("active")]
        public bool? Active { get; set; }

        [JsonProperty("createdBy")]
        public System.Guid CreatedBy { get; set; } // CreatedBy

        [JsonProperty("createdDate")]
        public System.DateTime CreatedDate { get; set; } // CreatedDate

        [JsonProperty("lastModifiedBy")]
        public System.Guid LastModifiedBy { get; set; } // LastModifiedBy

        [JsonProperty("lastModifiedDate")]
        public System.DateTime LastModifiedDate { get; set; } // LastModifiedDate

        [JsonProperty("transactionCategory")]
        public TransactionCategory TransactionCategory { get; set; }

        [JsonProperty("cashJournals")]
        public ICollection<CashJournal> CashJournal { get; set; }
    }
}

File: TreasurersApp/ClientApp/src/app/models/TransactionType.ts
export class TransactionType {
  id: number = null;
  transactionCategoryId: number = null;
  name: string = null;
  description: string = null;
  displayOrder: number = null;
  active = false;
  createdBy: string = null;
  createdDate: Date = null;
  lastModifiedBy: string = null;
  lastModifiedDate: Date = null;

  constructor(id: number,
    transactionCategoryId: number,
    name: string,
    description: string,
    displayOrder: number,
    active: boolean,
    createdBy: string,
    createdDate: Date,
    lastModifiedBy: string,
    lastModifiedDate: Date) {
    this.id = id;
    this.transactionCategoryId = transactionCategoryId;
    this.name = name;
    this.description = description;
    this.displayOrder = displayOrder;
    this.active = active;
    this.createdBy = createdBy;
    this.createdDate = createdDate;
    this.lastModifiedBy = lastModifiedBy;
    this.lastModifiedDate = lastModifiedDate;
  }
}
